fehm_toolkit/file_interface/avs.py

```diff
@@ -12,10 +12,10 @@
 }
 
 
-def read_avs(restart_file: Path) -> tuple[State, dict]:
```

Comment:
Changes in this file are slightly separate, just making them to keep the AVS reader up to date with the shift to `Decimal` values instead of `float`.